WordPress/openverse
# Add Free Music Archive to external sources in the frontend

## Fixes



Fixes #2486  by @zackkrida 

## Description



Added  `Free Music Archive` to the inbuilt `External sources` frontend functionality.


![Screenshot from 2024-02-27 18-03-06](https://github.com/WordPress/openverse/assets/70858293/02d49446-6544-4684-a7ae-efdd18df0b2f)

![image_screenshot_fma_test](https://github.com/WordPress/openverse/assets/70858293/e4f6edc9-f3fa-4e59-b449-f12a9aec0b59)
![Screenshot from 2024-02-27 18-03-39](https://github.com/WordPress/openverse/assets/70858293/7a06dfe7-28ba-4195-aa3b-e76fdf35c29f)



## Testing Instructions

_Optional:_ 
1. Run `just frontend/run dev`
2. Open an audio search page
3. Scroll down to the "External sources" button

The popup should contain "Free Music Archive", and clicking on it should give the results of the query on Openverse website.

## Checklist



[best_practices]:
  https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project#_commit_guidelines

## Developer Certificate of Origin


<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>
